plugin.video.playonbrowser/resources/language/English/strings.po

```diff
@@ -0,0 +1,78 @@
+# Playon Browser Media Center language file
```

Comment:
ah, I didn't catch that format change... thx for the info, I'll make the changes.